libraries/AP_HAL_Linux/VideoIn.cpp

```diff
@@ -303,6 +365,7 @@ bool VideoIn::_dequeue_frame(Frame &frame)
     frame.buf_index = buf.index;
     frame.timestamp = _timeval_to_us(buf.timestamp);
     frame.sequence = buf.sequence;
+    frame.bytesused = buf.bytesused;
```

Comment:
Regarding the use of this bytesused, I'd rather not put it in the videoin driver since it only depends on the use you want to have of the video frame. I'd rather keep it in the optical flow part.